docs/querying/api.md

```diff
@@ -382,6 +382,79 @@ $ curl http://localhost:9090/api/v1/alertmanagers
 }
 ```
 
+## Status
+
+Following status endpoints expose current Prometheus configuration.
+
+### Config
+
+The following endpoint returns currently loaded configuration file:
+
+```
+GET /api/v1/status/config
+```
+
+Currently, config is returned as dumped YAML file.
+
+```json
+$ curl http://localhost:9090/api/v1/status/config
+{
+  "status": "success",
+  "data": {
+    "yaml": "<content of the loaded config file in YAML>",
+  }
+}
+```
+
+### Flags
+
+The following endpoint returns flag values that Prometheus was configured with:
+
+```
+GET /api/v1/status/flags
+```
+
+All values are in a form of "string".
+
+```json
+$ curl http://localhost:9090/api/v1/status/flags
+{
+  "status": "success",
+  "data": {
+    "alertmanager.notification-queue-capacity": "10000",
+    "alertmanager.timeout": "10s",
+    "completion-bash": "false",
```

Comment:
This is not a flag of Prometheus, why is it here?

Reply:
That and things like `help-man` are probably coming in through the flag library.
Maybe that can be disabled – otherwise we can just blacklist those.

Reply:
Well.. This and same others (e.g `help`) are all kingpin flags, so they are included when we are dumping them into `flagsMap`.

UI `/flags` page shows them too. 
![image](https://user-images.githubusercontent.com/6950331/36434973-3f06293e-1658-11e8-95b2-36fc10a291a3.png)

To be consistent I think we should leave them, or if we want we can filter some out on both API and flag page. What do you think?

Reply:
@brian-brazil comment still valid